go/vt/vttablet/tabletserver/connpool/pool_test.go

```diff
@@ -126,9 +126,9 @@ func TestConnPoolSetCapacity(t *testing.T) {
 	defer connPool.Close()
 
 	assert.Panics(t, func() {
-		connPool.SetCapacity(-10)
+		_ = connPool.SetCapacity(context.Background(), -10)
 	})
-	connPool.SetCapacity(10)
+	_ = connPool.SetCapacity(context.Background(), 10)
```

Comment:
In tests, it's probably always worth requiring err to be nil. Who knows what bugs we might find that way?